pkgs/development/python-modules/msprime/default.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, pythonOlder
+, cython
+, gsl
+, numpy
+, newick
+, tskit
+, demes
+, pytestCheckHook
+, pytest-xdist
+}:
+
+buildPythonPackage rec {
+  pname = "msprime";
+  version = "1.2.0";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-YAJa2f0w2CenKubnYLbP8HodDhabLB2hAkyw/CPkp6o=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+    cython
+  ];
+
+  propagatedBuildInputs = [
+    gsl
+    numpy
+    newick
+    tskit
+    demes
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-xdist
+  ];
+
+  # Requires the module stdpopsim, not yet packaged
+  disabledTestPaths = [
+    "tests/test_demography.py"
+  ];
+
+  pythonImportsCheck = [
+    "msprime"
+  ];
+
+  meta = with lib; {
+    description = "Simulate genealogical trees and genomic sequence data using population genetic models";
+    homepage = "https://github.com/tskit-dev/msprime";
+    license = licenses.gpl3Only;
```

```suggestion
    license = licenses.gpl3Plus;
```
https://github.com/tskit-dev/msprime/blob/52abee1ed132ef18c0ad27c3b61be67e3f399a48/msprime/__init__.py#L11-L12